docs/user_guide/assessment.rst

```diff
@@ -501,13 +501,21 @@ argument. Here is an example of a pie chart.
 .. literalinclude:: ../auto_examples/plot_quickstart.py
     :language: python
     :start-after: # Customize plots with kind
+    :end-before: # Saving plots
 
 .. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_004.png
     :align: center
 
 There are many other customizations that can be done. More information can be found in
 :meth:`pandas.DataFrame.plot`.
 
+In order to save a plot, access the `matplotlib` Figure as below and save it with your
```

Comment:
We have the intersphinx mapping enabled for matplotlib. This should create a link to the matplotlib docs, but I'd recommend running it to check that it works. Sometimes it's a big finicky...
```suggestion
In order to save a plot, access the :class:`matplotlib.pyplot.Figure` as below and save it with your
```